src/stores/stores.ts

```diff
@@ -114,6 +116,7 @@ function getStoreSubstate(blocklyStoreProps: string[], tephraSimulationProps: st
       blocklyStore: pick(blocklyStoreProps)(blocklyStore),
       tephraSimulation: pick(tephraSimulationProps)(tephraSimulation),
       seismicSimulation: {},      // nothing to save yet
+      deformationSimulation: {},
```

Comment:
I don't think this needs to be here. These are serializations of the various MST stores, and we don't have a `deformationSimulation` store yet.

Reply:
Good catch - I had rolled back changes after our discussions earlier in the week and moved things to be part of the seismic simulation instead, but these managed to sneak past me.